logp/core_test.go

```diff
@@ -212,3 +216,117 @@ func TestLoggingECSFields(t *testing.T) {
 		}
 	}
 }
+
+func TestWithFileOutput(t *testing.T) {
+	var tempDir1, tempDir2 string
+	// Because of the way logp and zap work, when the test finishes, the log
```

Comment:
Add a reference to https://github.com/elastic/elastic-agent-libs/issues/179 so the next person to read this knows it can be removed.

Reply:
Done d55bae1